complete/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <script src="script.js"></script>
+  <meta name="description" content="Responsive Website">
+  <title>Responsive Website</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css">

```

Comment:
reset.css is `never` after the other stylesheets, it's always first, so then the stylesheets we (you) write can overwrite the reset.css values when needed

Reply:
reset.css, libraries and all other external assets need to be added before, so you can then code your own code on top of them